jss-sparsegl/script_for_plotting.R

```diff
@@ -0,0 +1,134 @@
+# only left plot_box() and libraries uncommendted if necessary
```

Comment:
1. Should uncomment everything
2. but source this in 1 chunk in the .Rmd with `eval = FALSE`

Reply:
At the beginning I do save everything in .Rmd to control which part should get evaluated when compiling but it didn't work as what I expected, so I switched to .R and this worked. Maybe I should try it again.